cloud-sql/postgres/activerecord/Gemfile

```diff
@@ -2,16 +2,17 @@ source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem "rails", "~> 5.2.3"
```

Comment:
Please restore the original expression `"~> 5.2.3"`.

Reply:
Done.